packages/core-data/src/queried-data/get-query-parts.js

```diff
@@ -73,6 +73,8 @@ export function getQueryParts( query ) {
 				// Example: Asking for titles in posts without title support.
 				if ( key === '_fields' ) {
 					parts.fields = getNormalizedCommaSeparable( value );
+					// Make sure to normalize value for `stableKey`
+					value = parts.fields.join();
```

Comment:
Nit: should we be using another variable to avoid changing the semantic of the original variable.

Reply:
I stuck with changing `value` so that we could leave the `parts.stableKey` builder alone - I thought it would be simpler this way. Is there enough of a drawback here to reconsider that?